package.json

```diff
@@ -63,5 +63,13 @@
         "extends": [
             "@commitlint/config-conventional"
         ]
+    },
+    "pnpm": {
+        "overrides": {
+            "react": "17.0.2",
+            "react-dom": "17.0.2",
+            "@types/react": "17.0.44",
+            "@types/react-dom": "17.0.15"
+        }
```

Comment:
I guess it doesn't work without those?

Reply:
Nope, see [build #3](https://jsadminbuilds.dev.cloud.coveo.com/job/plasma/job/PR-2718/3/)

Reply:
After the update to pnpm 7, it works without the overrides 👍

Reply:
🎉